tika/unpack.py

```diff
@@ -96,7 +97,10 @@ def _parse(tarOutput):
 
         contentMember = tarFile.getmember("__TEXT__")
         if not contentMember.issym() and contentMember.isfile():
-            content = _text_wrapper(tarFile.extractfile(contentMember)).read()
+            if version_info.major >= 3:
+                content = _text_wrapper(tarFile.extractfile(contentMember), encoding='utf8').read()
+            else:
+                content = tarFile.extractfile(contentMember).read().decode('utf8')
```

Comment:
I cannot tell you what the correct way to go about this is, but I don't believe this approach is it. There's no guarantee the file you're extracting is actually UTF-8 encoded - it could be anything.

Furthermore, `.decode('utf8')` can still fail and/or produce garbled output if the original encoding wasn't UTF-8. Does Tika have heuristics for encoding detection? If it does, you're likely better off feeding it the raw bytes unaltered and letting Tika handle this.